interpretation/deepseismic_interpretation/azureml_pipelines/dev/kickoff_train_pipeline.py

```diff
@@ -0,0 +1,8 @@
+"""
+Create pipeline and kickoff run
+"""
+from deepseismic_interpretation.azureml_pipelines.train_pipeline import TrainPipeline
+
+orchestrator = TrainPipeline("interpretation/deepseismic_interpretation/azureml_pipelines/pipeline_config.json")
+orchestrator.construct_pipeline()
+run = orchestrator.run_pipeline(experiment_name="DEV-train-pipeline")
```

Comment:
Would be nice to prefix experiment_name value with user's username and post-fix with a random GUID.

Reply:
Hm, I am not sure why we would want to do that. If we did do that, we would be running a separate experiment every run.

Reply:
@maxkazmsft

Reply:
Discussed might add some confusion so not including at this point. Can be picked up later